### ISSUE TITLE
cannot browse with screen reader between plugins in NPP (plugins -> plugin administrator)

### ISSUE THREAD

**Issue by gregjozk** [notepad-plus-plus/notepad-plus-plus#11600]
### Description of the Issue

as described in https://github.com/notepad-plus-plus/nppPluginList/issues/444 a dialog "plugin administrator" is not fully accessible for screen readers.

### Steps to Reproduce the Issue

1. start NVDA or simular screen reader.
2. start npp (I have installed 8.3.3)
3. go to plugins -> plugin administrator.
4. with "tab" key try to land on the list with plugins on any of three available tabs.

### Expected Behavior

with tab key focus should also land on a list with addons, such as it lands on a search field, some buttons etc.

### Actual Behavior

focus skips plugins list and lands only on search field, previous/next button, unnamed read only field, close button and place with three tabs.

### Debug Information

Notepad++ v8.3.3   (32-bit)
Build time : Mar 13 2022 - 17:11:10
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
OS Name : Windows 10 Enterprise (64-bit) 
OS Version : 2009
OS Build : 19044.1645
Current ANSI codepage : 1250
Plugins : DSpellCheck.dll mimeTools.dll NppConverter.dll NppExport.dll 

when I opened debug info, I found, that button "copy to clipboard" is also unavailable if I tried to find it only with "tab" key. there is only "ok" button.

thanks @chcg to redirect me.
